topics/admin/tutorials/systemd-supervisor/slides.html

```diff
@@ -2,16 +2,122 @@
 layout: base_slides
 logo: "GTN"
 
-title: "Controlling Galaxy with Supervisor"
+title: "Controlling Galaxy with systemd or Supervisor"
 contributors:
   - natefoo
+  - hexylena
 ---
-
 class: special
 
+# systemd
+
+The current Linux init system
+
+Used to bootstrap the user space and to manage system processes after booting
+
+---
+class: left
+
+# systemd
+
+Replaces:
+- `/etc/init.d/*`, `/etc/rc?.d`
+- `service` command
+- Upstart
+- `update-rc.d`
+
+INI config format
+
+---
+# systemd - Layout
+
+- `/lib/systemd/system` - System service definitions
+- `/etc/systemd/system` - User-defined service definitions
+
+---
+# systemd - Service file
+
+```ini
+[Unit]
+Description=Supervisor process control system for UNIX
+Documentation=http://supervisord.org
+After=network.target
+
+[Service]
+ExecStart=/usr/bin/supervisord -n -c /etc/supervisor/supervisord.conf
+ExecStop=/usr/bin/supervisorctl $OPTIONS shutdown
+ExecReload=/usr/bin/supervisorctl -c /etc/supervisor/supervisord.conf $OPTIONS reload
+KillMode=process
+Restart=on-failure
+RestartSec=50s
+
+[Install]
+WantedBy=multi-user.target
```

Comment:
Feels a bit funny to use supervisor here (and don't want to confuse them since you can do both systemd/supervisor, not one or the other), maybe galaxy would be a better option since people are familiar with it?

```suggestion
[Unit]
Description=A high performance web server and a reverse proxy server
Documentation=man:nginx(8)
After=network.target

[Service]
Type=forking
PIDFile=/run/nginx.pid
ExecStartPre=/usr/sbin/nginx -t -q -g 'daemon on; master_process on;'
ExecStart=/usr/sbin/nginx -g 'daemon on; master_process on;'
ExecReload=/usr/sbin/nginx -g 'daemon on; master_process on;' -s reload
ExecStop=-/sbin/start-stop-daemon --quiet --stop --retry QUIT/5 --pidfile /run/nginx.pid
TimeoutStopSec=5
KillMode=mixed

[Install]
WantedBy=multi-user.target
```

Reply:
Whoops, I meant to delete this slide, since I added the Galaxy service unit.